### PR TITLE
Created prototype web interface

### PR DESCRIPTION
**Pull request by Andy-ch**
http://ocr.ihor.andychweb.com/

### CODE REVIEW

File: ocr/webui/service.py [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                        help='Show "ps ux" command output on status queries',
+                        action='store_true')
+    parser.add_argument('-k',
+                        '--kill',
+                        help=('Use SIGKILL instead of SIGTERM '
+                              'when stopping the service'),
+                        action='store_true')
+    args = parser.parse_args()
+    command = args.command
+    getattr(sys.modules[__name__], command + '_output')(args)
```
Comment (Andy-ch) [inline ocr/webui/service.py]: This line executes a function corresponding to the command. The thing is, start_output function needs ACCESS_LOG_SIZE and ERROR_LOG_SIZE alone (other functions called here don't need them). When I tried passing those as parameters to all other functions too - pylint starts yelling at the unused variables. That is why I made them global
This makes me think that probably one day we will encounter situation when we will have to violate the code style rules for the sake of the functionality. And our current testing policy will interfere with that.

File: ocr/webui/service.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                  '--access-logfile access.log ' \
+                  '--log-file error.log ' \
+                  'main:APP'
+SERVICE_SHORT_NAME = '127.0.0.1:8001'
+if not os.path.exists('access.log'):
+    open('access.log', 'w').close()
+ACCESS_LOG_SIZE = os.stat('access.log').st_size
+if not os.path.exists('error.log'):
+    open('error.log', 'w').close()
+ERROR_LOG_SIZE = os.stat('error.log').st_size
```
Comment (Andy-ch) [inline ocr/webui/service.py]: Why are these global? See line 144

File: ocr/webui/service.py (hunk 1 above)
Comment (ferc255) [reply ocr/webui/service.py]: Probably I don't quite understand the issue, but can you declare those variables inside start_input? Or it causes errors?

**Comment by ferc255** [comment]
I've created pull request for your [Andy-ch/project](https://github.com/Andy-ch/project/pull/1) repository. I made your *__log_size_ variables local. I don't see any problems with pylint. Am I wrong?

File: ocr/webui/service.py (hunk 1 above)
Comment (ferc255) [reply ocr/webui/service.py]: I still don't see problems with functionality and pylint interaction. As you can see every issue is solvable.

**Comment by Andy-ch** [comment]
Approves anyone? :)

**Pull request merged**